### DISCUSSION TITLE
希望能ctrl+回车快捷键发送评论

### DISCUSSION THREAD

**Discussion by gcl-l** [the1812/Bilibili-Evolved#1654] [综合讨论 / General]
动态页和评论区用，用快捷键快速发送弹幕就更方便了。

**Comment by the1812** [comment]
之前有人提过了

弹幕按 enter

**Reply by gcl-l** [reply]
弹幕是可以快捷发。
我指的是视频评论区，跟动态评论区。缺少个快捷键发送的便携方式

**Reply by the1812** [reply]
https://github.com/the1812/Bilibili-Evolved/discussions/1538
https://github.com/the1812/Bilibili-Evolved/issues/1015

**Reply by gcl-l** [reply]
感谢回复，那行了
